orchestrator/graphql/schemas/process.py

```diff
@@ -61,36 +35,99 @@ class ProcessFormType:
 
 @strawberry.experimental.pydantic.type(model=ProcessStepSchema)
 class ProcessStepType:
-    stepid: strawberry.auto
+    step_id: strawberry.auto
     name: strawberry.auto
     status: strawberry.auto
     created_by: strawberry.auto
     executed: strawberry.auto
     commit_hash: strawberry.auto
     state: Union[JSON, None]
 
+    @authenticated_field(
+        description="Returns step id",
+        deprecation_reason="Changed to 'stepId' from version 1.2.3, removing after version 1.3.0",
+    )  # type: ignore
+    def stepid(self) -> Optional[UUID]:
+        return self.step_id
 
-@strawberry.experimental.pydantic.type(model=ProcessGraphqlSchema, directives=federation_key_directives)
+
+@strawberry.experimental.pydantic.type(model=ProcessSchema, directives=federation_key_directives)
 class ProcessType:
     process_id: strawberry.auto
+    product_id: strawberry.auto
+    customer_id: strawberry.auto
     workflow_name: strawberry.auto
     workflow_target: strawberry.auto
-    product: strawberry.auto
-    customer: strawberry.auto
     assignee: strawberry.auto
     failed_reason: strawberry.auto
     traceback: strawberry.auto
-    step: strawberry.auto
-    status: strawberry.auto
     last_step: strawberry.auto
+    last_status: strawberry.auto
     created_by: strawberry.auto
-    started: strawberry.auto
-    last_modified: strawberry.auto
+    started_at: strawberry.auto
+    last_modified_at: strawberry.auto
     is_task: strawberry.auto
     steps: strawberry.auto
     form: strawberry.auto
     current_state: Union[JSON, None]
 
+    @authenticated_field(
+        description="Returns process id",
+        deprecation_reason="Changed to 'processId' from version 1.2.3, removing after version 1.3.0",
+    )  # type: ignore
+    def pid(self) -> UUID:
+        return self.process_id
+
+    @authenticated_field(
+        description="Returns customer id",
+        deprecation_reason="Changed to 'customerId' from version 1.2.3, removing after version 1.3.0",
+    )  # type: ignore
+    def customer(self) -> UUID:
+        return self.customer_id
+
+    @authenticated_field(
+        description="Returns process workflow name",
+        deprecation_reason="Changed to 'workflowName' from version 1.2.3, removing after version 1.3.0",
+    )  # type: ignore
+    def workflow(self) -> str:
+        return self.workflow
```

Comment:
I think this should be

```suggestion
        return self.workflow_name
```